ocfweb/docs/docs/services/hpc.md

```diff
@@ -0,0 +1,65 @@
+[[!meta title="High Performance Computing"]]
+
+**NOTE: We are in the process of trialling this service to users so that we can make the service as accommodating and secure as possible. This means that items concerning the service, including this documentation, are subject to change. We will do our best to keep everyone updated and notified of changes as they come.**
+
+## Introduction
+
+At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. Our main HPC server is currently named `corruption`, however we also have plans to expand the cluster to maximize the use of the resources at our disposal.
```

Comment:
I don't think naming the server here is really important and might be confusing since they don't access it directly.